### PR TITLE
[change] Explosion component to use sprite

### PR DESCRIPTION
**Pull request by hakobpogh**

### CODE REVIEW

File: src/components/explosions/Explosion.module.scss [hunk 1]
```diff
@@ -1,20 +1,17 @@
 .Explosion {
+  --explosion-bg-x: calc(0px - var(--bg-double-block-size) * 16);
+  --explosion-bg-y: calc(0px - var(--bg-double-block-size) * 8);
+
   width: var(--double-block-size);
   height: var(--double-block-size);
-  background: center / contain;
-  animation: explosion 2s forwards;
+  background: black url("../../assets/images/sprite-big.png") no-repeat
+    /* background-position */ var(--explosion-bg-x) var(--explosion-bg-y);
```
Comment (MyasnikKhachkalyan) [inline src/components/explosions/Explosion.module.scss:8]: do we need this comment here?

**Review by MyasnikKhachkalyan — APPROVED** [review]

File: src/components/explosions/Explosion.module.scss (hunk 1 above)
Comment (hakobpogh) [reply src/components/explosions/Explosion.module.scss:8]: Not really. That's actually for readability.

Comment (hospogh) [reply src/components/explosions/Explosion.module.scss:8]: I think `background-position` property would fit here beter and it also descriptive :D

**Review by hospogh — APPROVED** [review]

File: src/components/explosions/Explosion.module.scss (hunk 1 above)
Comment (hakobpogh) [reply src/components/explosions/Explosion.module.scss:8]: That way you're adding to separate properties, and the browser adds some position first and then overrides it later. That's why smart editors suggest not doing it.

**Pull request merged**